buildpipeline/windows.groovy

```diff
@@ -21,7 +21,12 @@ simpleNode('windows.10.amd64.clientrs4.devex.open') {
         framework = '-allConfigurations'
     }
     else {
-        framework = "-framework:${params.TGroup}"
+        if (params.TGroup == 'netstandardTest') {
+            framework = "-framework:netcoreapp"
```

Comment:
Why would you chose netcoreapp? Isn't the point that you would test all the platforms?

Reply:
This is just building the product, so we pick netcoreapp to build the product to generate the targeting pack. If you check below, I change this again to netstandard for tests.

Reply:
Just to make sure, does that mean that `BuildingNETCoreAppVertical` is true?

Reply:
The way it was, it would for the source, but not for the tests but this is all going to change. I had a chat with @weshaggard and we decided to move this test leg to the build all configurations one.